packages/core/src/browser/label-parser.spec.ts

```diff
@@ -32,7 +32,7 @@ before(() => {
     statusBarEntryUtility = testContainer.get(LabelParser);
 });
 
-describe('StatusBarEntryUtility', () => {
+describe.only('StatusBarEntryUtility', () => {
```

Comment:
@cdamus please remove the `.only` or only this test in `@theia/core` will effectively run.

Reply:
Commit d17284f3 tabbar: support icon-less items fixes that and I've verified that the entire core suite (495 tests run and 2 skipped) is restored.